public/index.pug

```diff
@@ -146,14 +155,12 @@ html(lang="en-US")
                 class="grid-item__button"
                 href=`./icons/${icon.slug}.svg`
                 role="button"
+                data-icon=`${icon.slug}`
```

Comment:
You can actually do

```suggestion
                data-icon=icon.slug
```

which makes your intend a bit clearer (namely that the `data-icon` value should be exactly and only the icon's slug).